azdev/utilities/command.py

```diff
@@ -56,6 +55,28 @@ def cmd(command, message=False, show_stderr=True, **kwargs):
     except subprocess.CalledProcessError as err:
         return CommandResultItem(err.output, exit_code=err.returncode, error=err)
 
+def shell_cmd(command, message=False, stderr=None, stdout=None, check=True, raise_ex=True, timeout=None, 
+              executable=None):
+    # use default message if custom not provided
+    if message is True:
+        message = 'Running: {}\n'.format(command)
+    from . import display
+    if message:
+        display(message)
+
+    try:
+        subprocess.run(
+            command,
+            stdout=stdout,
+            stderr=stderr,
+            check=check,
+            timeout=timeout,
+            executable=executable,
+            shell=True)
+    except subprocess.CalledProcessError as err:
+        if raise_ex:
+            raise err
```

Comment:
It should be CLIError

Reply:
No

Reply:
For example, in venv.py line77, it is using raise_ex default setting. Will raise the error and terminate here in line78, the caller will not handler it.

Reply:
No, it absolutely should not be CLI error, this re-raises the call process error that we catch typically from subprocessor error. If you don't want that error, you set raise_ex to False and the CLI error is then raised.